lib/lf/uscalfe/lagr_fe.h

```diff
@@ -1295,12 +1296,22 @@ class FeLagrangeO3Tria final : public ScalarReferenceFiniteElement<SCALAR> {
   FeLagrangeO3Tria() = default;
   ~FeLagrangeO3Tria() override = default;
 
+  /** @copydoc ScalarReferenceFiniteElement::RefEl()
+   */
```

Comment:
strictly speaking, this is not necessary, because doxygen will automatically copy the documentation: https://craffael.github.io/lehrfempp/classlf_1_1uscalfe_1_1_fe_lagrange_o3_tria.html